core/Dockerfile

```diff
@@ -67,7 +67,7 @@ RUN RCFILE_PATH="/etc/blueosrc" \
     && echo "export GIT_DESCRIBE_TAGS=$GIT_DESCRIBE_TAGS" >> $RCFILE_PATH
 
 # create some Symlinks to make life easier
-RUN mkdir /shortcuts
+RUN mkdir -p /shortcuts
```

Comment:
has this ever happened? that is weird.
not a bad patch, I'm just wondering how it could happen.

Reply:
Yeah, I wonder, initially I thought it was some weird cache from GitHub, but it was also happening on my computer without cache :eyes: